docs/migration-guide.md

```diff
@@ -77,28 +96,10 @@ The end goal is for CDK to do all the work, and remove the YAML template:
 CDK -> cfn.json
 ```
 
-In the previous phase, we also created a [snapshot test](https://docs.aws.amazon.com/cdk/v2/guide/testing.html#testing_snapshot) (`cdk/lib/*.test.ts`),
-which we can treat as the source of truth, representing the exact resources in the stack. We can now begin the migration process.
-
-**Pro-tip:** Annotate the existing YAML template with inline comments as part of the migration and get it reviewed by your team - it might transpire that a resource is no longer needed and can be removed, meaning there's less to migrate!
-
-As a first step, it is a good idea to read through the template to understand if you have any [stateful resources](stateful-resources.md), as these require a bit of care.
-
-When migrating stateful resources:
-1. Identify a resource(s) to move from YAML to CDK.
-2. Write the CDK and comment out/delete the resource(s) from the YAML template.
-3. Run the tests:
-   - If the changes are acceptable, update the snapshot.
-   - If the changes are not acceptable, refactor CDK and re-run the tests.
-4. With the smallest change made, we can now raise a PR, merge it, and move to the next resource(s).
-
-That is, we can use the snapshot in a [red/green refactor cycle](https://blog.cleancoder.com/uncle-bob/2014/12/17/TheCyclesOfTDD.html).
-We'll repeat this process until the YAML template is empty and can be removed.
+In order to achieve this, you should now follow the specific migration guide for the pattern that you're moving to:
 
-Please also read the specific guides for:
-- [for EC2 apps](./migration-guide-ec2.md)
-- [for Lambdas which run on a schedule/cron](./migration-guide-scheduled-lambda.md)
-- [for an API which serves requests using AWS Lambda](./migration-guide-api-with-lambda.md)
+- [for `GuEc2App`](./migration-guide-ec2.md)
+- [for `GuApiLambda` or `GuApiGatewayWithLambdaByPath`](./migration-guide-api-with-lambda.md)
+- [for `GuScheduledLambda`](./migration-guide-scheduled-lambda.md)
 
-:information_source: Migration guides will be coming soon for other architectures.
-Please contact DevX if you'd like to discuss the migration of an application which uses an architecture that is not listed above.
+If your pattern is not listed here then please contact the DevX Reliability & Operations team for support.
```

Comment:
I'm a bit confused by what "your pattern" means here. Is the idea here to point out that if you intend to use a pattern for which we don't have a guide (i.e. GuPlayWorkerApp) you should reach out for support? Or is it more if the app you're migrating uses an architecture for which we don't have a pattern?